Framework/Muon/src/CalculateMuonAsymmetry.cpp

```diff
@@ -203,6 +203,26 @@ void CalculateMuonAsymmetry::exec() {
     API::MatrixWorkspace_sptr normWS =
         API::AnalysisDataService::Instance().retrieveWS<API::MatrixWorkspace>(
             wsNames[j]);
+
+    normWS->mutableY(0) = ws->y(0) / norms[j];
```

Comment:
I think we should split off the workspace transform into its own function and we can call it twice. Then when we fix the missing constant we just need to edit one place.